sass-precompiling.md

```diff
@@ -0,0 +1,292 @@
+---
+---
+
+# SASS precompiling
+
+## Table of contents
+
+- [Enable SASS precompiling](#add-sass-precompiling)
+- [Additional functionalities](#additional-functionalities)
+- [The same workflow for mobile and web](#the-same-workflow-for-mobile-and-web)
+
+## Add SASS precompiling
+
+We will now see how to add SASS files precompiling to the build process. First we need to add to our `package.json` the following dependency:
+
+```javascript
+"devDependencies": {
+  "gulp-sass": "*"
+}
+```
+ 
+First set a variable that stores the source and the destination folders in the `gulpfile.js`:
```

Comment:
we have already used `first` :wink: `First we need to add to our `package.json` `